build/images/scripts/start_ovs

```diff
@@ -109,6 +109,8 @@ SLEEP_PID=
 
 function quit {
     log_info $CONTAINER_NAME "Stopping OVS before quit"
+    # sleep 2 seconds to wait antrea-agent shutdown for Flexible-IPAM
```

Comment:
Nit: as the feature is going to be renamed to AntreaIPAM we might want to use this name throughout this PR (including commit message)

Reply:
Fixed: Flexible IPAM -> AntreaIPAM